lib/elixir/lib/option_parser.ex

```diff
@@ -118,6 +125,36 @@ defmodule OptionParser do
   end
 
   @doc """
+  The same as `parse/2` but raises an `OptionParser.InvalidOptionError`
+  exception if any invalid options are given.
```

Comment:
I would make this summary shorter without mentioning the kind of error that is raised (we can mention that in the body of the doc):

> The same as `parse/2`, but raises an exception if there are invalid options.

This is similar to the documentation of other bang variants such as [`File.cd!/2`](http://elixir-lang.org/docs/stable/elixir/File.html#cd!/2):

> The same as `cd/1`, but raises an exception if it fails.